7.8.вложенные_циклы.часть_1/7.8.9.py

```diff
@@ -0,0 +1,5 @@
+n = int(input())
+for i in range(1, n + 1):    
+    for j in range(i):     
```

Comment:
Вот тут j не используется:
```suggestion
    for _ in range(i):     
```